test/react-native/features/fixtures/performance-scenarios/scenarios/ManualSpanScenario.js

```diff
@@ -2,25 +2,24 @@ import React, { useEffect } from 'react'
 import { SafeAreaView, View, Text, StyleSheet } from 'react-native'
 import BugsnagPerformance from '@bugsnag/react-native-performance'
 
-export const ManualSpanScenario = (endpoint, apiKey) => {
-  const App = () => {
-    useEffect(() => {
-      BugsnagPerformance.start({ apiKey, endpoint, maximumBatchSize: 1, appName: 'com.bugsnag.reactnative.performance', appVersion: '1.2.3' })
-      BugsnagPerformance.startSpan('ManualSpanScenario').end()
-    }, [])
+export const config = {
+  maximumBatchSize: 1,
+  autoInstrumentAppStarts: false,
+  appVersion: '1.2.3'
+}
```

Comment:
definitely neater - but are you still starting BugsnagPerformance?

Reply:
yep - BugsnagPerformance is now started in `ScenarioLauncher.js`, before the scenario is loaded.

this allows us to better test the app start instrumentation as the call to start sets a wrapper component provider which then wraps our scenario component